tests/cli/pipeline-spec.yaml

```diff
@@ -44,3 +44,23 @@ verbose-logs-with-sleep:
       log_line()
       time.sleep(.1)
       log_line()
+
+custom-formatters:
+  pipeline:
+  - run: load_resource
+    parameters:
+      url: ../data/datapackage.json
+      resource: my-spiffy-resource
+  - run: duplicate
+    parameters:
+      source: my-spiffy-resource
+      target-name: my-spiffy-xlsx-resource
+      target-path: my-spiffy-resource.xlsx
+  - run: dump.to_path
+    parameters:
+      out-path: custom_formatters
+      force-format: false
+      counters:
+        resource-hash: ''
+      file-formatters:
+        xlsx: [custom_formatters, XLSXFormat]
```

Comment:
I think it would work better here if we used a string instead of a list (`custom_formatters.XLSXFormat`). Wdyt?

Reply:
sure, and then split on the last dot ?